### PR TITLE
NONE: add coef for page size for builds

### PR DESCRIPTION
**Pull request by bgvozdev**
**What's in this PR?**
Increaseing page size for builds: a sync is going on forever for one particular customer

**Why**
I'd like to see how the changed page size affects the speed of the sync. 

**Added feature flags**
- increase-builds-page-size-coef

### CODE REVIEW

File: src/sync/build.ts [hunk 1]
```diff
@@ -25,17 +27,27 @@ const getTransformedBuilds = async (workflowRun, gitHubInstallationClient, logge
 export const getBuildTask = async (
 	logger: Logger,
 	gitHubInstallationClient: GitHubInstallationClient,
-	_jiraHost: string,
+	jiraHost: string,
 	repository: Repository,
-	cursor: string | number = 1,
-	perPage?: number
+	cursorOrigStr: string | number = 1,
```
Comment (rachellerathbone) [inline src/sync/build.ts:32]: Nit: I can see we're now using cursor and perPage in the function block but is there any other names we could use here? perPageOrig, cursorOrigStr, cursorOrig aren't super clear and it gets a little hard keeping track of what's what. I'm okay with letting this through now because it's an experiment but would probably be good to update these for readability sakes if we decide to keep this in the code.

File: src/sync/sync.types.ts [hunk 2]
```diff
@@ -14,9 +14,9 @@ export interface TaskProcessors {
 		gitHubInstallationClient: GitHubInstallationClient,
 		jiraHost: string,
 		repository: Repository,
-		cursor?: string | number,
-		perPage?: number,
-		messagePayload?: BackfillMessagePayload
+		cursor: string | number | undefined,
```
Comment (rachellerathbone) [inline src/sync/sync.types.ts:17]: What's the reason for this change? I don't think cursor is a required arg. There are multiple places in the code where we are setting the values for cursor/perPage if it isn't passed e.g. in `getBuildTask` for cursor -> `cursor: string | number = 1`

File: src/sync/build.ts (hunk 1 above)
Comment (bgvozdev) [reply src/sync/build.ts:32]: I'm open for suggestions :)

File: src/sync/sync.types.ts (hunk 2 above)
Comment (bgvozdev) [reply src/sync/sync.types.ts:17]: Just typing @rachellerathbone , I'm removing optionality: we always pass over perPage and messagePayload

Comment (bgvozdev) [reply src/sync/sync.types.ts:17]: also, typescript would've yelled if something was wrong, however tests are green ...

Comment (rachellerathbone) [reply src/sync/sync.types.ts:17]: lol you put a lot of faith in our tests.

**Review by rachellerathbone — APPROVED** [review]
